packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -112,6 +116,7 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, Lockable {
         uint32 priceRequestTime;
         uint256 proposerBond;
         uint256 finalFee;
+        uint32 liveness;
```

Comment:
@nicholaspai I think most of the changes in this are good/worthwhile (blocking speed up after liveness, comment clarifications), but I think this liveness one adds unnecessary complexity to the implementation for something that doesn't seem to break anything or have any major downsides (by my estimation). Thoughts on dropping that part?

Reply:
yeah it should